ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -36,18 +40,87 @@ def __init__(self):
 
         self.final_waypoints_pub = rospy.Publisher('final_waypoints', Lane, queue_size=1)
 
+
         # TODO: Add other member variables you need below
 
         rospy.spin()
 
     def pose_cb(self, msg):
         # TODO: Implement
+        #
+        # rate=rospy.Rate(10)
+        # while not rospy.is_shutdown():
+
+        #     print("current pose")
+        #     print(msg.pose.position.x)
+        #     print(msg.pose.position.y)
+        #     print(msg.pose.position.z)
+
+        #     rate.sleep()
+        self.pose=msg.pose.position
+        self.posex=msg.pose.position.x
```

Comment:
is it enough to only keep self.pose? why need to have self.pose and self.posex at the same time?